tox.ini

```diff
@@ -3,9 +3,20 @@ envlist= py38,style,bandit
 
 [testenv]
 usedevelop= true
+
 extras= test
-# astropy will complain if the home directory is missing
-passenv= HOME
+
+passenv =
+    TOXENV
+    CI
+    CODECOV_*
+    HOME
+    CRDS_*
```

Comment:
Why do we need `CRDS_*` and the others that follow?  We're not passing any of them in from the actions workflow...

Reply:
But they do get defined in the `jwst.datamodels` tests, though maybe we don't need these defined as this only is needed if they are passed in from env vars outside the tox/pytest run itself?